notebooks/regression/tutorial_regression.md

```diff
@@ -234,7 +261,230 @@ All strategies except the Naive one give effective coverage close to the expecte
 0.95 value (recall that alpha = 0.05), confirming the theoretical garantees.
 
 
-## 2. Estimating the epistemic uncertainty of out-of-distribution data
+## 2. Estimating the aleatoric uncertainty of heteroscedastic noisy data
+
+```python
+import warnings
+warnings.filterwarnings("ignore")
```

Comment:
These two lines should have been deleted by gsed ?

Reply:
Let's see if it shows up again when I make a clean doc next time.